tests/test_cli.py

```diff
@@ -143,12 +141,7 @@ def test_pyver_is_deprecated(capsys):
     """
     pkgs = os.path.join(_get_test_root(), "data/cli/nominal_release.yml")
     repo = os.path.join(_get_test_root(), "data/cli/nominal_repository.yml")
-    locs = os.path.join(_get_test_root(), "data/cli/locations.yml")
-    cmd = (
-        f"{pkgs} {repo} --prefix pfx --release rel "
-        f"--locations-config {locs} "
-        f"--pyver 3.8"
-    )
+    cmd = f"{pkgs} {repo} --prefix pfx --release rel " f"--pyver 3.8"
```

Comment:
Don't really need string concatenation here.

Reply:
I don't understand your comment, could you please make a suggestion on change required on your view?

Reply:
I think @kwinkunks refers to joining those two strings: 
`cmd = f"{pkgs} {repo} --prefix pfx --release rel --pyver 3.8"`

Reply:
I see, thanks for pointing out. Fixed.